kge/job/eval.py

```diff
@@ -141,3 +146,66 @@ def create_from(
             new_config.set("eval.split", eval_split, create=True)
 
         return super().create_from(checkpoint, new_config, dataset, parent_job)
+
+
+class TrainingLossEvaluationJob(EvaluationJob):
+    """ Evaluating simply by using the training loss """
+
+    def __init__(self, config: Config, dataset: Dataset, parent_job, model):
+        super().__init__(config, dataset, parent_job, model)
+
+        training_loss_eval_config = config.clone()
+        training_loss_eval_config.set("console.quiet", False)
```

Comment:
Why is this done? Quit should be repsected, not changed.

Reply:
Got it from the old PR, thought it was discussed already. I guess if we want to simply evaluate a model, we don't need the output from the internal training job

Reply:
I think we do, but if we don't then quiet should be set to True, right?